src/pineko/evolve.py

```diff
@@ -178,6 +181,7 @@ def evolve_grid(
     # Here we are checking if the EKO contains the rotation matrix (flavor to evol)
     elif not np.allclose(operators.rotations.inputpids, br.rotate_flavor_to_evolution):
         raise ValueError("The EKO is neither in flavor nor in evolution basis.")
+    # do it
```

Comment:
?

Reply:
Well after the checks you start "doing it" :). Joke apart I can remove it

Reply:
[9b34b1d](https://github.com/NNPDF/pineko/pull/75/commits/9b34b1d9e94671b5000428c78f7fb013a20d2913)